cap/modules/mail/utils.py

```diff
@@ -22,257 +22,98 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-import re
+from flask import render_template
 
-from flask import current_app
-from flask_login import current_user
-from flask_principal import RoleNeed
-
-from invenio_accounts.models import User
-from invenio_access.permissions import Permission
-
-from cap.modules.mail.tasks import create_and_send
+CONFIG_DEFAULTS = {
```

Comment:
Maybe we should update the structure to a per action key/value. More centralised per action, if it grows